Tests/js/views/assets/ez-languageswitcherview-tests.js

```diff
@@ -0,0 +1,366 @@
+
+            this.view = new Y.eZ.LanguageSwitcherView({
+                container: '.container',
+                content: this.contentMock,
+                location: this.locationMock,
+                languageCode: this.languageCode,
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should provide empty other translations array to the template": function () {
+            var origTpl;
+
+            origTpl = this.view.template;
+            this.view.template = function (vars) {
+                Assert.isArray(vars.otherTranslations, "The template should receive an array with other translations");
+                Assert.areEqual(
+                    0, // minus current language version
+                    vars.otherTranslations.length,
+                    "The number of other translations should be one less than number of content translations"
+                );
+                return origTpl.apply(this, arguments);
+            };
+            this.view.render();
+        },
+    });
+
+    eventTest = new Y.Test.Case({
+        name: "eZ Language Switcher View event test",
+
+        setUp: function () {
+            this.location = {
+                id: 'kazimierz-deyna'
+            };
+            this.languageCode = 'eng-GB';
+            this.versionInfo = {
+                'languageCodes': 'eng-GB,pol-PL,ger-DE,fre-FR'
+            };
+            this.contentMock = new Mock();
+            this.locationMock = new Mock();
+
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: this.location
+            });
+            Mock.expect(this.contentMock, {
+                method: 'get',
+                args: ['versionInfo'],
+                returns: this.versionInfo
+            });
+
+            this.view = new Y.eZ.LanguageSwitcherView({
+                container: '.container',
+                content: this.contentMock,
+                location: this.locationMock,
+                languageCode: this.languageCode,
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should expand the view when language switcher link is tapped": function () {
+            var container = this.view.get('container'),
+                that = this,
+                otherTranslationsLink;
+
+            this.view.set('expanded',false);
+            this.view.render();
+            otherTranslationsLink = container.one('.ez-dropdown-list-indicator'),
+
+            otherTranslationsLink.simulateGesture('tap', function () {
+                that.resume(function () {
+                    Assert.isTrue(
+                        this.view.get('expanded'),
+                        'The expanded should state should be true'
+                    );
+                });
+            });
```

Comment:
Actually after updating npm dependencies and bower my test says the same so I'll dig into it bit more.